docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -68,6 +68,14 @@ Malware protection also allows you to manage a blocklist to prevent specified ap
 extending the list of processes that {elastic-defend} considers malicious. Use the **Blocklist enabled** toggle
 to enable or disable this feature for all hosts associated with the integration policy. To configure the blocklist, refer to <<blocklist>>.
 
+When *Detect* malware is enabled, {agent} will quarantine any malicious file it finds. Specifically {agent} will remove it from it's current location, obfuscate it (XOR encrypt it with the encryption key `ELASTIC`), move it to a different folder, and rename it as a string like `318e70c2-af9b-4c3a-939d-11410b9a112c`.
+
+- On MacOS, all {agent} versions move quarantined files to the folder `/System/Volumes/Data/.equarantine`.
+- On Linux, all {agent} versions move quarantined files to the folder `.equarantine` at the root of the mount point of the file being quarantined.
+- On Windows, {agent} versions 8.5 and later move quarantined files to the folder `[DriveLetter:]\.quarantine`, unless they are from the `C:` drive, in which case it moves them to `C:\Program Files\Elastic\Endpoint\state\.equarantine`. {agent} versions earlier than 8.5 move quarantined files to `[DriveLetter:]\.quarantine` regardless of which drive they were originally on.
```

Comment:
This last bullet is pretty dense, so adding the line break might improve readability. Your call!

```suggestion
- On Windows, {agent} versions 8.5 and later move quarantined files to the folder `[DriveLetter:]\.quarantine`, unless they are from the `C:` drive. These files are moved to `C:\Program Files\Elastic\Endpoint\state\.equarantine`. 
+
{agent} versions earlier than 8.5 move quarantined files to `[DriveLetter:]\.quarantine`, regardless of their original drive.
```